docs/integrations/logfire.md

```diff
@@ -0,0 +1,164 @@
+# Logfire
+
+[Logfire](https://docs.pydantic.dev/logfire/), a new tool from Pydantic, is built on OpenTelemetry. As Mirascope is also built on Pydantic, it's appropriate for us to ensure seamless integration with them.
+
+## How to use Logfire with Mirascope
+
+```python
+pip install logfire
+
+from mirascope.integrations.logfire import with_logfire
+```
+
+`with_logfire` is a decorator that can be used on all Mirascope classes to automatically log both Mirascope calls and also all our [supported LLM providers](https://docs.mirascope.io/latest/SUPPORTED_PROVIDERS/).
+
+## Examples
+
+### Call
+
+A Mirascope call:
+
+```python
+import logfire
+
+from mirascope.core import anthropic, prompt_template
+from mirascope.integrations.logfire import with_logfire
+
+logfire.configure()
+
+def format_book(title: str, author: str):
+    return f"{title} by {author}"
+
+@with_logfire
+@anthropic.call(model="claude-3-5-sonnet-20240620", tools=[format_book])
+@prompt_template("""Recommend a {genre} book.""")
```

Comment:
this can be a normal string, doesn't need to be a docstring

Reply:
done